src/processes/scoring/calc-karma-score.js

```diff
@@ -73,8 +80,8 @@ const calcKarmaScore = async (userId, sign_user_score = null) => {
   }
   const combined_user_score = await combinedUserScore(userId, userScoreCol, sign_user_score);
   const percentileScore = await percentileUserScore(combined_user_score, userScoreCol);
-  const karma_score = (KARMA_SCORE_MULTIPLIER * percentileScore) ** KARMA_SCORE_EXPONENT;
-  await setUserKarmaScore(userId, combined_user_score, karma_score, userScoreCol);
+  const karma_score = KARMA_SCORE_MULTIPLIER * percentileScore ** KARMA_SCORE_EXPONENT;
+  await setUserKarmaScore(userId, combined_user_score, karma_score, userScoreCol, percentileScore);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation of `karma_score` has been changed. It now uses exponentiation on `percentileScore` before multiplying by `KARMA_SCORE_MULTIPLIER`. This change seems logical if you want to give more weight to the percentile score in the karma score calculation. However, be aware that this could significantly alter the distribution of karma scores, especially if percentile scores are not evenly distributed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->